app/controllers/tournaments_controller.rb

```diff
@@ -20,6 +42,18 @@ def scores
   end
 
   def success
+    unless Tournament.exists?(params[:id])
+      render json: { error: "Tournament Not Found" }, status: :not_found
```

Comment:
you can just `return` here, that way you don't need an else, decreasing the indentation for the rest of the function